src/main/scala/edu/ie3/simona/model/participant/control/QControl.scala

```diff
@@ -169,19 +169,21 @@ object QControl {
       *   The function
       */
     override def activeToReactivePowerFunc(
-        sRated: Power,
+        sRated: ApparentPower,
         cosPhiRated: Double,
         nodalVoltage: Dimensionless,
     ): Power => ReactivePower = { activePower: Power =>
       val qMaxFromP = Megavars(
         sqrt(
-          pow(sRated.toMegawatts, 2) -
+          pow(sRated.toActivePower(1.0).toMegawatts, 2) -
```

Comment:
Wouldn't `sRated.toMegavoltamperes` be the actual solution?

Reply:
Yes, this was also my soltion.